File: apps/family-service/src/modules/device/device.controller.ts
import { Controller } from '@nestjs/common';
import { ApiTags } from '@nestjs/swagger';
import { DeviceService } from './device.service';

@ApiTags('device')
@Controller('api/v1/device')
export class DeviceController {
  constructor(private readonly deviceService: DeviceService) {}
}

File: apps/identity-service/src/main/java/com/family/device/controller/DeviceController.java
package com.family.device.controller;

import io.swagger.v3.oas.annotations.tags.Tag;
import org.springframework.web.bind.annotation.*;

@Tag(name = "Devices", description = "IoT device registry")
@RestController
@RequestMapping("/api/v1/devices")
public class DeviceController {
}
